plugins/node/opentelemetry-instrumentation-mysql2/test/mysql.test.ts

```diff
@@ -161,7 +165,11 @@ describe('mysql@2.x', () => {
 
         query.on('end', () => {
           const spans = memoryExporter.getFinishedSpans();
-          assert.strictEqual(spans[0].name, sql);
+          assert.strictEqual(spans[0].name, 'SELECT');
+          assert.strictEqual(
+            spans[0].attributes[SemanticAttributes.DB_STATEMENT],
+            query
```

Comment:
What is the type of this variable? Is this an object or a string?

Reply:
It's a string. Code is now updated